executor/executor.go

```diff
@@ -797,32 +793,51 @@ func (e *CheckTableExec) Next(ctx context.Context, req *chunk.Chunk) error {
 
 	// The number of table rows is equal to the number of index rows.
 	// TODO: Make the value of concurrency adjustable. And we can consider the number of records.
-	concurrency := 3
-	wg := sync.WaitGroup{}
-	for i := range e.srcs {
-		wg.Add(1)
-		go func(num int) {
-			defer wg.Done()
+	if len(e.srcs) == 1 {
+		err := e.checkIndexHandle(ctx, e.srcs[0])
+		if err != nil {
+			return errors.Trace(err)
+		}
+		return nil
```

Comment:
Hint: `return e.checkIndexHandle(ctx, e.srcs[0])` save several lines of code

Reply:
Thank. fixed